CHANGELOG.md

```diff
@@ -54,6 +54,7 @@ Bottom level categories:
 #### Naga
 
 - When emitting GLSL, Uniform and Storage Buffer memory layouts are now emitted even if no explicit binding is given. By @cloone8 in [#7579](https://github.com/gfx-rs/wgpu/pull/7579).
+- Diagnostic rendering methods (i.e., `naga::{front::wgsl::ParseError,WithSpan}::emit_error_to_string_with_path`) now accept more types for their `path` argument via a new sealed `AsDiagnosticFilePath` trait. By @atlv24 and @ErichDonGubler in [#7643](https://github.com/gfx-rs/wgpu/pull/7643).
```

Comment:
i think its probably worth putting bushRAT in here too

Reply:
An excellent call-out. Added!